### ISSUE TITLE
Sign In With Google Temporarily disabled for this app

### ISSUE THREAD

**Issue by nrcarnevali** [phw198/OutlookGoogleCalendarSync#915]
----

**OGCS Version**: v1.2.3.4  
**Installed or Portable**: Portable

**Problem Description**  
Google Wont allow login

**Steps to Reproduce the Issue**  
Started app - tried beta and alpha and same issue

**Any other information** 

 
![G2](https://user-images.githubusercontent.com/23621316/70221112-8ceaf700-16fc-11ea-9489-17d6d2968f9d.PNG)

When googling it seems this is something that has to be fixed by the dev

Thanks in advance!

**Comment by phw198** [comment]
> Has to be fixed by the dev

Yes and no. You're basically on one of the API quotas that is being run down, as Google haven't approved it yet (a new process recently introduced by the overlords). There's nothing to stop you continuing to use it - just that you need to click on the "Proceed anyway" link off the bottom of your screenshot.

I can assure you that this quota app is configured in _exactly_ the same way as the others that Google have approved :slightly_smiling_face:

**Comment by nrcarnevali** [comment]
Hi Paul,

Thanks for the reply - but perhaps I am missing something. There is no approve or proceed anyway link..

![image](https://user-images.githubusercontent.com/23621316/70451862-00716900-1a5b-11ea-89e7-4504c75046a9.png)

**Comment by nrcarnevali** [comment]
Forgot to add.. I am also seeing this as well with a different app (MailCal) I use to sync calendars on my phone (in case my laptop is off and OGCS is not running).

**Comment by phw198** [comment]
Ah yes, apparently as that API quota has not yet been verified, Google only allow up to 100 users to _start_ using it (even though there are lots of people still using it already) - otherwise they block it entirely.

Go to the `Settings` > `Google` > `Disconnect Account` and then go through the OAuth process from scratch - you should go onto a different, authorised API app...

**Comment by nrcarnevali** [comment]
Thanks - That worked perfectly.

Thanks,
Nick

**Comment by MW65** [comment]
Unfortunately , disconnecting account didn't work for me . Have tried reconnecting many times but continue to receive and can't get past Error Message: " Sign in with Google temporarily disable for this app etc " .. It would seem looking at the dates on this thread that the denied Google permission is no longer temporary , but permanent. Shame - I rather liked the app.

**Comment by phw198** [comment]
@MW65 The OGCS app is still very much verified and approved by Google, but I can't tell why you might be receiving this error without your OGcalsync.log file.
